plane/source/docs/quadplane-parameters.rst

```diff
@@ -29,6 +29,37 @@ Key Parameters
    throttle while flying or you risk disarming your motors.
 -  The default :ref:`SCHED_LOOP_RATE<SCHED_LOOP_RATE>` for a QuadPlane is to 300 (Hz). Most QuadPlanes do not need this to be rasied. Some very small vehicles (< 1Kg) might benefit from setting it to 400. In heavier vehicles, their higher inertia results in lower effective control response rates, so they do not benefit from a higher loop rate. Raising above 300 only leads to larger log files in these vehicles.
 
+.. _return_behavior_setup:
+
+Return Behavior QuickSetup Guide:
+=================================
+
+While there are many parameters setting distances and heights for the various home return modes/behaviors (read each mode's wiki page), this is a quick setup guide for basic behaviors.
+
+1.If you lose RC link for greater than :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` in fixed wing flight set :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` value below to obtain various behavior options:
+- "0" : do nothing if in AUTO mode, otherwise switch to RTL mode
+- "1" : switch to RTL mode
```

Comment:
This should mention the Q_RTL_MODE parameter which will mean (if I understand correctly) that even in fixed wing flight, a long failsafe will actually QRTL if enabled (value > 0).

Reply:
unless an option to  switch to RTL is selected that(q_rtl_mode) has no impact....further down implications of RTL mode for a QPlane are discussed...will add note referencing #3,4 below this